sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -29,7 +29,7 @@ namespace Azure.Security.KeyVault.Certificates.Tests
     public partial class CertificateClientLiveTests : CertificatesTestBase
     {
         public CertificateClientLiveTests(bool isAsync, CertificateClientOptions.ServiceVersion serviceVersion)
-            : base(isAsync, serviceVersion, null /* RecordedTestMode.Record /* to re-record */)
+            : base(isAsync, serviceVersion, RecordedTestMode.Record /* to re-record */)
```

Comment:
re-comment the `RecordTestMode`

Reply:
🙄 We really have to figure out a better way. Env vars require reloading VS (if using VS, which for test runs is superior) and code changes get forgotten. Maybe some file: if empty, record all; otherwise, parse out (fully-qualified?) class names to record, maybe even methods. /cc @pakrym #justthinkingoutloud

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/14176

Reply:
You should use the RecordedTestAttribute. That way you don't have to switch anything at all :)

Reply:
For GA'd code, if a test fails I'd rather investigate why than re-record, which 1) requires provisioning resources (I don't leave them around because it's expensive), and 2) either will fail, or record the wrong result - either way a waste of time if I just made a breaking change.

Reply:
Good point. Wonder if we can add a button to the test explore toolbar that would run with re-record?

Reply:
Re-recorded tests still fail, so nothing would be silently re-recorded. If you don't have resources provisioned, the failure will also still occur as it would have.